django_slack_oauth/models.py

```diff
@@ -18,8 +15,23 @@ class SlackUser(models.Model):
     def is_slacked(self):
         return self.access_token
 
-    def __unicode__(self):
-        return unicode(self.slacker)
+    def __str__(self):
+        return self.slacker.__str__()
 
     class Meta:
         db_table = 'slack_user'
+
+
+class SlackOAuthRequest(models.Model):
```

Comment:
Agree about the naming. Just afraid for users about sudden stop of writes into `slack_user` :) On the other hand there was no any major release so version could be 1.0.0 with release notes about updates.